src/libs/Notification/LocalNotification/types.ts

```diff
@@ -10,8 +10,19 @@ type LocalNotificationData = {
 type LocalNotificationModule = {
     showCommentNotification: (report: Report, reportAction: ReportAction, onClick: LocalNotificationClickHandler) => void;
     showUpdateAvailableNotification: () => void;
-    showModifiedExpenseNotification: (report: Report, reportAction: ReportAction, onClick: LocalNotificationClickHandler) => void;
+    showModifiedExpenseNotification: (params: LocalNotificationModifiedExpenseParams) => void;
     clearReportNotifications: ClearReportNotifications;
 };
 
-export type {LocalNotificationModule, LocalNotificationClickHandler, LocalNotificationData};
+type LocalNotificationModifiedExpenseParams = {
+    report: Report;
+    reportAction: ReportAction;
+    onClick: LocalNotificationClickHandler;
+    movedFromOrToReportMessage: string | undefined;
```

Comment:
Probably I'm missing something, do we want `movedFromOrToReportMessage` to be explicitly added? because we can make it optional.
```
movedFromOrToReportMessage?: string;
```
and when passing the object we don't add it